Dockerfile

```diff
@@ -1,7 +1,6 @@
 # Build Agora from source
```

Comment:
I know what you intended as you explained. But, please refer to [the example of a simple PR](https://github.com/bpfkorea/agora/pull/130)

In addition, it would be good to add the issue number in the pull request's description like this.
`Related to #825` or `Fixes #825`
Please refer to other PRs.

Thanks!